src/components/Notification/Notification.stories.mdx

```diff
@@ -0,0 +1,95 @@
+import { Meta, Story, Preview, Props } from '@storybook/addon-docs/blocks';
+import { withKnobs, text, select, boolean } from '@storybook/addon-knobs';
+import Notification from '../Notification';
+import PresentComponent from '../storyUtils/PresentComponent';
+import Stack from '../storyUtils/Stack';
+
+<Meta title="Design System/Notification" component={Notification} />
+
+# Notification
+
+# Usage
+
+```js
+import { Notification } from '@orfium/ictinus';
+
+<Notification />;
+```
+
+# Props
+
+<Props of={Notification} />
+
+# Inline Notification
+
+Inline Notification
+
+<Preview>
+  <Story name="Inline Notifiction" parameters={{ decorators: [withKnobs] }}>
+    <Stack>
+      <PresentComponent name="" width={768}>
+        <Notification
+          message={text('info', 'Informative Message')}
```

Comment:
use the same name as prop e.g `message={text('message', 'Informative Message')}`

We will avoid confusions from devs